vllm/executor/ray_utils.py

```diff
@@ -229,12 +229,17 @@ def initialize_ray_cluster(
             the default Ray cluster address.
     """
     assert_ray_available()
-
     # Connect to a ray cluster.
     if is_hip() or is_xpu():
-        ray.init(address=ray_address,
-                 ignore_reinit_error=True,
-                 num_gpus=parallel_config.world_size)
+        # Try to connect existing ray instance and create a new one if not found
```

Comment:
I am afraid this may break hip path. can you keep hip coda path and add a `is_xpu()` branch?

Reply:
I think this is a common error despite of hip or xpu. It's only about parameter acceptance for ray `init` call.